app/assets/javascripts/models/beta/projectBoard.js

```diff
@@ -68,44 +68,72 @@ export const toggleColumn = (projectBoard, column, callback) => {
 }
 
 // Drag And drop utils
+const calculateNewPosition = (aboveStory, bellowStory, storyState) => {
+  const aboveStoryState = aboveStory?.state;
+  const bellowStoryState = bellowStory?.state;
+  const aboveStoryPosition = aboveStory?.newPosition;
+  const bellowStoryPosition = bellowStory?.newPosition;
+
+  if (!aboveStory) return 1;
```

Comment:
Should we move this guard to the first line of the function? It seems if the `aboveStory` is absent we don't even need to collect those variables above.

Reply:
You are right, I will change. I had placed it with the other returns for the sake of organization.